app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -213,6 +213,7 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
     // model attribute with AWS.
     vm.cloudVolumeModel.aws_volume_type = data.volume_type;
     vm.cloudVolumeModel.aws_availability_zone_id = data.availability_zone.ems_ref;
+    //vm.cloudVolumeModel.aws_cloud_volume_snapshot_id = data.cloud_volume_snapshot_id;
```

Comment:
The snapshot id should be set here so that, when editing the volume, the snapshot will be displayed. The ID should be returned by the current API call.